From 0aaf1ea1247f9821df75908d3343a0efe43cb3ac Mon Sep 17 00:00:00 2001
From: Ben Walton <bwalton@opencsw.org>
Date: Sat, 27 Nov 2010 15:25:31 +0100
Subject: [PATCH] Update path in sample hook file

The post-receive-email sample hook file referenced /usr/share.  Update
this to be /opt/csw/share.

Signed-off-by: Ben Walton <bwalton@opencsw.org>
---
 templates/hooks--post-receive.sample |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/templates/hooks--post-receive.sample b/templates/hooks--post-receive.sample
index 7a83e17..aaa5121 100755
--- a/templates/hooks--post-receive.sample
+++ b/templates/hooks--post-receive.sample
@@ -12,4 +12,4 @@
 # see contrib/hooks/ for a sample, or uncomment the next line and
 # rename the file to "post-receive".
 
-#. /usr/share/doc/git-core/contrib/hooks/post-receive-email
+#. /opt/csw/share/doc/git/contrib/hooks/post-receive-email
-- 
1.7.3

